tensorflow_probability/python/mcmc/internal/util_test.py

```diff
@@ -196,8 +196,28 @@ def fn(x, y):
 @test_util.test_all_tf_execution_regimes
 class SmartForLoopTest(test_util.TestCase):
 
-  def test_python_for_loop(self):
+  @parameterized.parameters(0, 1)
```

Comment:
Why split this test into two, rather than testing on 0, 1, 10?  The expected number of calls then becomes `max(1, loop_iters)`, does it not (except for JAX and Eager mode, as usual)?

Reply:
That is the expected behavior unless the `loop_iters=0`, which is probably the test case I wanted to isolate.